speaker_info/35b2c544-660e-401e-b503-0e14c635303a/metas.json

```diff
@@ -1,3 +1,10 @@
 {
-  "supported_features": { "permitted_synthesis_morphing": "NOTHING" }
-}
+    "supported_features": {},
+    "range": [
+        {
+            "style_id": 8,
+            "low": 5.07,
+            "high": 6.5
+        }
+    ]
+}
```

Comment:
### データを配置する場所について

これはエンジンのmetas.jsonに書いてるけど、制作の都合上コアのmetas.jsonに書く方がやりやすいかもしれません。
データセットからピッチ範囲を計算する方針の場合は特に。
あとコアに書けばコアライブラリを使っている人も使えて便利かもしれません。
どうするかわからないけど、将来的にそういう設計に変えるかも、という共有です。

<details><summary>english</summary>

### About where to place the data

This is written in the engine's metas.json, but it might be easier to handle if written in the core's metas.json, especially if we're calculating the pitch range from the dataset.
Also, if it's written in the core, it might be useful for people using the core library.
I'm not sure how to go about it, but it's a comment that we might change to such a design in the future.

</details>

Reply:
同意です。ENGINE の `metas.json` ではなく CORE から取得すべきだと考えます。  
なぜならピッチ範囲情報と並列関係にある `SpeakerStyle` の属性値（`name` / `id` / `type`）が CORE から取得する情報であるからです。